embed-api-v3.rst

```diff
@@ -0,0 +1,119 @@
+:doc:`Back in June </hoverxref-intersphinx>` we announced
+intersphinx support in `sphinx-hoverxref`_ 0.6b1,
+which allowed users to include modals and tooltips
+also for cross references to other projects.
+
+.. figure:: img/sphinx-hoverxref-showcase.gif
+   :width: 60%
+   :align: center
+   :alt: sphinx-hoverxref displaying tooltips for cross references
+   :target: /_images/sphinx-hoverxref-showcase.gif
+
+   sphinx-hoverxref displaying a tooltip including an equation
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io/
+
+Embed API v3 and sphinx-hoverxref 1.0
+-------------------------------------
+
+Even though some projects were already using it,
+the extension used older versions of our Embed API,
+and by that time we already had a plan to overcome some of its limitations.
+Among other things, we wanted the Embed API to
+
+- Support embedding content from pages hosted outside Read the Docs, and
+- Avoid coupling with Sphinx internal format,
+  and directly parse the HTML instead.
+
+After some months of work, **we are excited to publish v3 of our Embed API,
+and with it, version 1.0 of sphinx-hoverxref**.
+
+To test the API directly, you can do
+
+.. code-block:: console
+
+   $ curl -s "https://readthedocs.org/api/v3/embed/\
+   > ?url=https://docs.readthedocs.io/en/latest/features.html\
+   > %23read-the-docs-features" | jq
+   {
+     "url": "https://docs.readthedocs.io/en/latest/features.html#read-the-docs-features",
+     "fragment": "read-the-docs-features",
+     "content": "<div class=\"section\" id=\"read-the-docs-features\">\n<h1>Read the Docs ...",
+     "external": false
+   }
+
+Or visually explore the query in the `web interface`__ instead.
+
+.. __: https://readthedocs.org/api/v3/embed/?url=https://docs.readthedocs.io/en/latest/features.html%23read-the-docs-features
+
+Making the backend do the request has several advantages:
+
+- It can return to the client only the fragment of interest, instead of
+  the whole page, reducing processing times in the browser.
+- It can cache the result, thanks to the generous support of CloudFlare.
+
+Some big projects already using it include `Tweepy <https://docs.tweepy.org/>`_,
+a Python client for Twitter, and `Scrapy <https://docs.scrapy.org/>`_,
+a framework for crawling websites.
+
+Current limitations
+-------------------
+
+We would like to invite the community to try out these features and
+send us feedback. In particular, the current version still has some limitations
+that we would like to address soon:
+
+- For security reasons, we have an allowlist of external domains
+  the Embed API works with. We would like to invite the community
+  to suggest more domains to add there, and evaluate them case by case.
+- The JavaScript embed client is included in the assets of `sphinx-hoverxref`_,
+  but it is not yet available as a standalone library that can be reused
+  with standard frontend packaging tools.
```

Comment:
Ending on the limitations of the tool feels odd. We should at least have a call to action to try out the new tools!